### DISCUSSION TITLE
Popconfirm for Tailwind

### DISCUSSION THREAD

**Discussion by cyggarro** [mdbootstrap/TW-Elements#1530] [Share ideas & request features]
This component should provide a confirmation dialog when an action is triggered by the user, such as deleting a record or submitting a form. 

The dialog should include the ability to customize the message displayed to the user, as well as the text for the confirm and cancel buttons.

**Comment by filipkappa** [comment]
Thanks for submitting this idea!

It gathered so much traction that we have added a task for this feature: https://github.com/mdbootstrap/Tailwind-Elements/issues/1559

You can track our progress live here:
https://tailwind-elements.com/docs/standard/getting-started/changelog/

**Comment by filipkappa** [comment]
We added this to Tailwind Elements! You can now use the feature:
https://tailwind-elements.com/docs/standard/components/popconfirm/
